Haskell/tests/syntax_test_cabal.cabal

```diff
@@ -0,0 +1,329 @@
+-- SYNTAX TEST "Packages/Haskell/Cabal.sublime-syntax"
+
+cabal-version:   >=2.2
+-- <- meta.mapping.key.cabal entity.name.tag.cabal
+-- ^^^^^^^^^^ meta.mapping.key.cabal
+--           ^ meta.mapping.cabal
+--            ^^^^^^^^^ meta.mapping.value.version.cabal
+-- ^^^^^^^^^^ entity.name.tag.cabal
+--           ^ punctuation.separator.mapping.key-value.cabal
+--               ^^ keyword.operator.comparison.cabal
+--                 ^^^ meta.number.version.cabal constant.numeric.value.cabal
+--                  ^ punctuation.separator.decimal.cabal
+
+name:            pandoc
+-- <- meta.mapping.key.cabal entity.name.tag.cabal
+-- ^ meta.mapping.key.cabal
+--  ^ meta.mapping.cabal
+--   ^^^^^^^^^^^^^^^^^^^ meta.mapping.value.other.cabal
+-- ^ entity.name.tag.cabal
+--  ^ punctuation.separator.mapping.key-value.cabal
+
+version:         2.11.3.1
+-- <- meta.mapping.key.cabal entity.name.tag.cabal
+-- ^^^^ meta.mapping.key.cabal
+--     ^ meta.mapping.cabal
+--      ^^^^^^^^^^^^^^^^^^ meta.mapping.value.version.cabal
+-- ^^^^ entity.name.tag.cabal
+--     ^ punctuation.separator.mapping.key-value.cabal
+--               ^^^^^^^^ meta.number.version.cabal constant.numeric.value.cabal
+--                ^ punctuation.separator.decimal.cabal
+--                   ^ punctuation.separator.decimal.cabal
+--                     ^ punctuation.separator.decimal.cabal
+
+author: Jon Doe <jon.doe@nowhere.com>
```

Comment:
Freudian slip? This guy's name is John. Jon is the other guy.

Reply:
Could also be Foo Bar. It's a test case. Content doesn't matter.

Reply:
it was ... a joke, nvm